Tests/IGListAdapterTests.m

```diff
@@ -313,6 +313,21 @@ - (void)test_whenCellsExtendBeyondBounds_thatVisibleCellsExistForSectionControll
     XCTAssertEqual([self.adapter visibleCellsForSectionController:sectionController6].count, 0);
 }
 
+- (void)test_whenCellsExtendBeyondBounds_thatVisibleIndexPathsExistForSectionControllers {
+    self.dataSource.objects = @[@2, @3, @4, @5, @6];
+    [self.adapter reloadDataWithCompletion:nil];
+    id sectionController2 = [self.adapter sectionControllerForObject:@2];
+    id sectionController3 = [self.adapter sectionControllerForObject:@3];
+    id sectionController4 = [self.adapter sectionControllerForObject:@4];
+    id sectionController5 = [self.adapter sectionControllerForObject:@5];
+    id sectionController6 = [self.adapter sectionControllerForObject:@6];
+    XCTAssertEqual([self.adapter visibleIndexPathsForSectionController:sectionController2].count, 2);
+    XCTAssertEqual([self.adapter visibleIndexPathsForSectionController:sectionController3].count, 3);
+    XCTAssertEqual([self.adapter visibleIndexPathsForSectionController:sectionController4].count, 4);
+    XCTAssertEqual([self.adapter visibleIndexPathsForSectionController:sectionController5].count, 1);
+    XCTAssertEqual([self.adapter visibleIndexPathsForSectionController:sectionController6].count, 0);
+}
```

Comment:
@Malecks np, apologies that some of the `starter-task`s can have a bit more to them under the surface 😄 ! 

Off the top of my head:

- Copy [this test](https://github.com/Instagram/IGListKit/blob/master/Tests/IGListStackSectionControllerTests.m#L399-L417)
- Assert the actual `NSIndexPath` values (not just counts) for each child section controller

Hopefully that helps!